### PR TITLE
[SIG-1856] - Message template alert

### PR DESCRIPTION
**Pull request by janjaap**
This PR contains changes that show an alert when a default text with template characters is used to submit an incident's changed status.

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js [hunk 1]
```diff
@@ -90,11 +90,17 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
...
   handleSubmit = e => {
     e.preventDefault();
-    this.props.onPatchIncident({
-      id: this.props.incident.id,
-      type: 'status',
-      patch: { status: { state: this.form.value.status, text: this.form.value.text } },
-    });
+    const { value } = this.form;
+
+    if (value.text.match(/{{[^}]+}}/gi) !== null) {
```
Comment (spaces-ga) [inline src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]: I would suggest a RegEx like this: `/(?:{{|}})/gi`

And replace the `.match()` for `.test()`

```js
if (value.text.test(/(?:{{|}})/gi)
```

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.test.js [hunk 2]
```diff
@@ -214,5 +214,21 @@ describe('<StatusForm />', () => {
         type: 'status',
       });
     });
+
+    it.only('should show an alert when the text contains template characters', () => {
```
[the remark this answers is not in the document]
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.test.js]: Whoops. Good catch

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]: Indeed, much simpler.

**Comment by janjaap** [comment]
@spaces-ga Comments addressed

**Pull request merged**